packages/shared/src/domain/address/state.ts

```diff
@@ -0,0 +1,123 @@
+export function normalizeState(state: string): USState | undefined {
+  if (Object.values(states).includes(USState[state as keyof typeof USState])) {
+    return USState[state as keyof typeof USState];
+  } else if (states[state]) {
+    return states[state];
+  } else if (state === "DC") {
+    return USState.DC;
+  }
+  return undefined;
+}
+
+export function normalizeStateStrict(state: string): USState {
+  const stateOrUndefined = normalizeState(state);
+  if (!stateOrUndefined) throw new Error("Invalid state.");
+  return stateOrUndefined;
+}
+
+export enum USState {
```

Comment:
Anyway to get this from the SDK? It creates a circular dependency. Maybe I remove from SDK?

Reply:
Yeah, with the SDK importing from shared, it'd be good to have it only on shared.

Reply:
Done.

Reply:
@raf the issue now is that we no longer export the USState enum on the sdk -- is this breaking functionality for our users? I can do something like `export const USState = { ...USStateShared };` on api-sdk to keep the idea there.

Reply:
Good catch, and I think its good to export it there from shared, yeah

Reply:
Added back as const